pkg/doc/vc/vcutil/sdjwt.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vcutil
+
+import (
+	"crypto"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/sdjwt/common"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+)
+
+func ReplaceCredentialSubjectDisclosures(credentialSubject interface{}, disclosures []string) error {
```

Comment:
Util function for replacing digests with disclosures values. Can be a part of AFGO

Reply:
We already have this/similar functionality in verifier in AFGO and the Verifier in AFGO already returns verified claims(with values). Currently this function is not referenced  in your pull request. We can discuss why it is needed and then expose it as separate function in AFGO.

Reply:
As aligned - into the next PR I'll mentioned func